typescript-version/src/@core/layouts/components/vertical/navigation/index.tsx

```diff
@@ -116,15 +116,16 @@ const Navigation = (props: Props) => {
       <Box sx={{ height: '100%', position: 'relative', overflow: 'hidden' }}>
         {/* @ts-ignore */}
         <ScrollWrapper
-          containerRef={(ref: any) => handleInfiniteScroll(ref)}
+          
           {...(hidden
             ? {
                 onScroll: (container: any) => scrollMenu(container),
                 sx: { height: '100%', overflowY: 'auto', overflowX: 'hidden' }
               }
             : {
                 options: { wheelPropagation: false },
-                onScrollY: (container: any) => scrollMenu(container)
+                onScrollY: (container: any) => scrollMenu(container),
+                containerRef={(ref: any) => handleInfiniteScroll(ref)}
```

Comment:
I think you mean

`containerRef=(ref: any) => handleInfiniteScroll(ref)`

Reply:
This is embarrassing, you're right